WORKSPACE

```diff
@@ -37,9 +37,9 @@ bind(
 # Determine SHA256 `wget https://github.com/istio/envoy/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # envoy commit date: 09/10/2019
 # bazel version: 0.28.1
-ENVOY_SHA = "8647faf091fa51b86037e448d75ca6bc29a4d23f"
+ENVOY_SHA = "81a490a66eb44f43adc34f81d1ef0b2cfbdb55e5"
```

Comment:
@jplevyak Also update the same SHA in `istio.deps`. We have not got rid of that file yet in `release-1.3`.

Reply:
done